File: atelier_couture/src/data/entites/Article.java
package data.entites;

public class Article extends AbstractEntity {
    protected String libelle;
    protected int qteStock;
    protected String photo;
    protected Categorie categorie;

    public String getLibelle() {
        return libelle;
    }

    public void setLibelle(String libelle) {
        this.libelle = libelle;
    }

    public int getQteStock() {
        return qteStock;
    }

    public void setQteStock(int qteStock) {
        this.qteStock = qteStock;
    }

    public String getPhoto() {
        return photo;
    }

    public void setPhoto(String photo) {
        this.photo = photo;
    }

    public Categorie getCategorie() {return categorie;}

    public void setCategorie(Categorie categorie) {this.categorie = categorie;}

    @Override
    public String toString() {
        return "Article{" +
                "libelle='" + libelle + '\'' +
                ", qteStock=" + qteStock +
                ", photo='" + photo + '\'' +
                '}';
    }
}

File: model/Article.php
<?php

class Article{
    protected int $id=0;
    protected string $libelle;
    protected int $qteStock;
    protected string $categorie;
    protected string  $photo;

    public function getLibelle()
    {
        return $this->libelle;
    }


    public function setLibelle($libelle)
    {
        $this->libelle = $libelle;

        return $this;
    }


    public function getQteStock()
    {
        return $this->qteStock;
    }


    public function setQteStock($qteStock)
    {
        $this->qteStock = $qteStock;

        return $this;
    }


    public function getPhoto()
    {
        return $this->photo;
    }


    public function setPhoto($photo)
    {
        $this->photo = $photo;

        return $this;
    }


    public function getCategorie()
    {
        return $this->categorie;
    }


    public function setCategorie(string $categorie)
    {
        $this->categorie = $categorie;

        return $this;
    }


    public function getId()
    {
        return $this->id;
    }


    public function setId($id)
    {
        $this->id = $id;

        return $this;
    }
}